.rubocop_todo.yml

```diff
@@ -0,0 +1,11 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2019-06-25 14:24:35 +0200 using RuboCop version 0.32.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
```

Comment:
Perhaps it's better to fix this case or add an ignore in the file itself? Does that work?

Reply:
Fixed by Excluding the check in rubocop.yml as directly in the file doesn't work (to late?).